common/src/main/java/org/schemarepo/validation/AllValidator.java

```diff
@@ -0,0 +1,44 @@
+package org.schemarepo.validation;
+import org.schemarepo.SchemaEntry;
+import org.schemarepo.SchemaValidationException;
+import org.schemarepo.Validator;
+import org.schemarepo.config.Config;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.Map;
+
+/**
+ * <p>
+ * A {@link Validator} for validating the provided schema against all
+ * schemas in the Iterable in {@link #validate(String, Iterable)}.
+ * </p>
+ * <p>
+ * Uses the {@link ValidationStrategy} provided in the constructor to
+ * validate the Schema against each Schema in the Iterable, in Iterator
+ * order, via {@link ValidationStrategy#validate(String, String)}.
+ * </p>
+ */
+public final class AllValidator implements Validator {

```

Comment:
I'm not married to any names. You tell me. I don't really care, I just used what already existed in the Avro validation code.